css/logo-nav.css

```diff
@@ -680,6 +680,115 @@ body {
     background-color: white;
 }
 
+@media(max-width:509px) {
```

Comment:
509px  is  not an already existing transition. Please use already existing values.

Reply:
So you just want me to take that `max-width` out altogether?